src/core-recipe.jl

```diff
@@ -87,8 +87,12 @@ function Makie.plot!(p::TopoPlot)
         return
     end
     notify(p.resolution) # trigger above (we really need `update=true` for onany)
-
-    padded_data = lift(pad_data, p.data, npositions, p.pad_value)
+    
+    if !isempty(methods(to_value(p.enlarge_value)))
```

Comment:
calling `methods` explicitly is a bit weird. What do you want to do here?

Reply:
find out whether elarge_value is a function or not. This is what I googled how to find out whether a variable is callable or not. I guess, one could also check whether it is a Number, and else call the function. probably easier to understand